code/controllers/subsystem/vote.dm

```diff
@@ -473,12 +473,12 @@ SUBSYSTEM_DEF(vote)
 
 // We also need to remove our action from the player actions when we're cleaning up.
 /datum/action/vote/Remove(mob/removed_from)
-	if(removed_from.client)
-		removed_from.client?.player_details.player_actions -= src
+	if(removed_from.persistent_client)
+		removed_from.persistent_client.player_actions -= src
```

Comment:
```suggestion
	removed_from.persistent_client?.player_actions -= src
```
any reason to not do this instead?

Reply:
there is an else statement, this if has meaning.